manifests/init.pp

```diff
@@ -11,11 +11,16 @@
   $ca_crt,
   $cert,
   $key,
+  $ta            = undef,
   $dh            = undef,
   $mode          = 'client', # client or server
+  $protocol      = 'udp',
+  $cypher        = undef, # Specify specific CYPHER if required.
```

Comment:
Cypher was chosen as a specific difference from cipher which is used in the configuration file.
The variable name cypher is used constantly in manifest and template.